config/config.go

```diff
@@ -109,6 +109,17 @@ var (
 	DefaultMarathonSDConfig = MarathonSDConfig{
 		RefreshInterval: Duration(30 * time.Second),
 	}
+
+	// The default Kubernetes SD configuration
+	DefaultKubernetesSDConfig = KubernetesSDConfig{
+		NodeMetricsPort: 10255,
+		ScrapeKey:       "prometheus.io/scrape",
```

Comment:
What do you mean? These are annotations on the kubernetes service to be able to configure the targets via discovery. They do that by setting labels on the targets (relabelling) during discovery phase, but this needs to be configurable to allow certain kubernetes providers to change this key if they want to.